File: iteration1/Lecturer.java

public class Lecturer extends Person {

    private String lecturerID;

    public String getLecturerID() {
        return lecturerID;
    }

    public Lecturer(String name, String lastName, String lecturerID) {
        super(name, lastName);
        this.lecturerID = lecturerID;
    }


    public Lecturer() {


    }

    @Override
    public String toString() {
        return "Lecturer: " + this.getFirstName() + " " + this.getLastName();
    }


}

File: iteration3/Lecturer.py
from Staff import Staff


class Lecturer(Staff):

    __lecturer_id = ""

    def __init__(self, name, last_name, lecturer_id):
        super().__init__(name, last_name)
        self.__lecturer_id = lecturer_id

    def __str__(self):
        return f"Lecturer: {self.get_first_name()} {self.get_last_name()}"
